core/storage/classifier/gae_models.py

```diff
@@ -181,6 +181,24 @@ def create(
         training_job_instance.put()
         return instance_id
 
+    @classmethod
+    def query_training_jobs(cls, cursor):
+        """Generates a unique id for the training job of the form
+        {{exp_id}}.{{random_hash_of_16_chars}}
+
+        Args:
+            cursor: str. It represent the position of cursor in the query
+            result.
+        Returns:
+            List of the ClassifierTrainingJobModels with status new or pending.
+        """
+        query = cls.query(cls.status.IN([
+            feconf.TRAINING_JOB_STATUS_NEW,
+            feconf.TRAINING_JOB_STATUS_PENDING]))
```

Comment:
I'm a bit confused. Why are we fetching pending jobs? I thought "pending" means they're already being processed by some VM.